### PR TITLE
[RW-6633] Add bcftools to All of Us

### PR DESCRIPTION
**Pull request by ericsong**

### CODE REVIEW

File: terra-jupyter-aou/Dockerfile [hunk 1]
```diff
@@ -110,6 +110,15 @@ RUN curl -SL --output dotnet.tar.gz "https://dotnetcli.azureedge.net/dotnet/Runt
 
 RUN echo "Sys.setenv(RETICULATE_PYTHON = '$(which python3)')" >> ~/.Rprofile
 
+RUN wget https://github.com/samtools/bcftools/releases/download/1.12/bcftools-1.12.tar.bz2 && \
```
Comment (rtitle) [inline terra-jupyter-aou/Dockerfile:113]: No strong preference really, but wondering if this should be done from a tmp directory like `/tmp/bcftools`?  (Or maybe `/usr/local/share`?)

Comment (calbach) [reply terra-jupyter-aou/Dockerfile:113]: +1 (see other tmpdir handling above for examples)

File: terra-jupyter-aou/Dockerfile [hunk 2]
```diff
@@ -110,6 +110,15 @@ RUN curl -SL --output dotnet.tar.gz "https://dotnetcli.azureedge.net/dotnet/Runt
 
 RUN echo "Sys.setenv(RETICULATE_PYTHON = '$(which python3)')" >> ~/.Rprofile
```
Comment (calbach) [inline terra-jupyter-aou/Dockerfile:111]: nit: would probably put this below the bcftools install, just to keep package setup somewhat separate from environment configuration

File: terra-jupyter-aou/Dockerfile (hunk 1 above)
Comment (ericsong) [reply terra-jupyter-aou/Dockerfile:113]: sounds good

**Review by calbach — APPROVED** [review]

**Comment by rtitle** [comment]
Change LGTM but looks like the `Test terra-jupyter-aou / test_docker_image` job failed. Do we know if this is a known issue? It would be good to fix the image or the test if needed.

**Comment by ericsong** [comment]
I think it may be unrelated to my changes actually - judging from the same failure on this no op PR https://github.com/DataBiosphere/terra-docker/pull/218

**Comment by rtitle** [comment]
OK - we can look into it separate then I think

**Review by rtitle — APPROVED** [review]

**Comment by ericsong** [comment]
I spent the past several hours looking into this but wasn't able to reproduce or make much headway.. I think I'm a little too unfamiliar with the build process to debug it well. At most, I was able to gather that it probably is due to the tensorflow upgrade to 2.5.0. I believe tensorflow packages are compiled with a specific CUDA version in mind and it may be throwing an error as a result. I was surprised to see that tensorflow was installed at all given that we specify tensorflow-cpu as the install target; maybe due to keras pulling in tensorflow as a dependency?

I think I'm going to merge my PR and request that this issue be looked into separately. I believe our image should be OK since it's based off the terra-jupyter-gatk:1.1.2 image which is already built and presumably doesn't have the version incompatibility issue.

**Pull request merged**